### DISCUSSION TITLE
Date of release of the next version

### DISCUSSION THREAD

**Discussion by Brnoch** [skia4delphi/skia4delphi#166] [General]
Hi,
Just a question, but no pressure about it :-) : do you have any idea of the date of release of the next version of skia4delphi ?
Best regards,
Bruno

**Comment by Brnoch** [comment]
And another one: do you think that the current code of the next version is enough stable to be used with the vcl ?

**Reply by viniciusfbb** [reply]
Yes, it will be.

**Comment by viniciusfbb** [comment]
Today. It will be major version 4.0.0 and there will be some compatibility breaks as we will update the Skia version, so we will have some changes in the api.

**Comment by Brnoch** [comment]
Great news ! I didn't imagine that it would be released the same day as my message :-)
Thanks a lot.

**Reply by viniciusfbb** [reply]
See our [new release v4.0.0](https://github.com/skia4delphi/skia4delphi/releases)
